qemu/overlay

```diff
@@ -0,0 +1 @@
+overlay.normal
```

Comment:
why change the directory to `overlay.normal` and link to it with overlay?

Reply:
During the development i usually create multiple copies of `overlay` that has different binaries and setups. I normally never push those other `overlay` directories. but over time it became a hassle to keep renaming the directories and know which ones has what.

I eventually decided to just create a symlink `overly` and suffix each directory with a name. 
```sh
overlay.normal # probably should have named it `overlay.default`
overlay.bootstrap 
#etc...
overlay -> overlay.normal
```

this way i can switch by just changing the symlink to which overlay i want to use